src/pages/workspace/tags/WorkspaceViewTagsPage.tsx

```diff
@@ -234,6 +235,17 @@ function WorkspaceViewTagsPage({route}: WorkspaceViewTagsProps) {
                     cancelText={translate('common.cancel')}
                     danger
                 />
+                <View style={[styles.pv4, styles.ph5]}>
+                    <ToggleSettingOptionRow
+                        title={translate('common.required')}
+                        switchAccessibilityLabel={translate('common.required')}
+                        isActive={Boolean(policyTagList?.required)}
```

Comment:
NAB. `currentPolicyTag` and `policyTagList` are the same. Let's remove `policyTagList`

Reply:
https://github.com/Expensify/App/pull/42972/commits/53a95143bf6e9977ff21a6958ee6de116e3c8e66